lib/stat_tracker.rb

```diff
@@ -1,7 +1,10 @@
 require 'csv'
+require_relative 'game_team'
+require_relative 'game'
+require_relative 'team'
 
 class StatTracker
-  attr_reader :games_path, :teams_path, :game_teams_path
```

Comment:
Didn't need to hang onto the path, much more important to hold onto the games, teams and game_teams (seriously, there needs to be a better term for this)